filebrowser.theme.inc

```diff
@@ -127,7 +127,7 @@ function theme_dir_listing_list_view($node) {
             $visible_columns[$column_name] = TRUE;
             $unsorted_rows[$file_name][FILEBROWSER_DATA_NAME_ICON] = array(
                 'data' => _filebrowser_thumbnails_generate($node, $data),
-                'style' => 'width:1%;'
+                'class' => 'icon'
```

Comment:
Array items should end with a comma:
```suggestion
              'class' => 'icon',
```

Also, the indentation is more than it should be - needs to be reduce by two spaces (also the line with the other array item above this line).